src/publisher.rs

```diff
@@ -57,12 +59,12 @@ pub extern "C" fn z_publisher_options_default() -> z_publisher_options_t {
 
 /// An owned zenoh publisher.
 ///
-/// Like most `z_owned_X_t` types, you may obtain an instance of `z_X_t` by loaning it using `z_X_loan(&val)`.  
-/// The `z_loan(val)` macro, available if your compiler supports C11's `_Generic`, is equivalent to writing `z_X_loan(&val)`.  
+/// Like most `z_owned_X_t` types, you may obtain an instance of `z_X_t` by loaning it using `z_X_loan(&val)`.
```

Comment:
Those double trailing spaces have meaning in markdown: they indicate a line-break that doesn't create a new paragraph, I'll reinsert them for you :)

Reply:
Ah nice, I didn't know that. My editor automatically removes trailing white spaces. Thanks for fixing this